spec/index.bs

```diff
@@ -6350,6 +6378,13 @@ GPUQueue includes GPUObjectBase;
 
 {{GPUQueue}} has the following methods:
 
+<dl dfn-type=attribute dfn-for=GPUQueue>
+    : <dfn>lastSignaledValue</dfn> of type {{GPUSignalValue}}, readonly
+    ::
+        The smallest signal value representing the progress of all the work that has been submitted
```

Comment:
Ah sorry, this was meant to be the last value returned by the call to `GPUQueue.signal()`. We should add a `completedValue` as well that's like what you currently get by calling `GPUFence.getCompletedValue()`.